lib/request.coffee

```diff
@@ -82,7 +79,7 @@ module.exports = getRequest = ({
 			if authorizationHeader?
 				options.headers.Authorization = authorizationHeader
 
-			if not isEmpty(options.apiKey)
+			if typeof options.apiKey == 'string' and options.apiKey.length > 0
```

Comment:
I'm forgetting CoffeeScript a bit, but should this be

```suggestion
			if typeof options.apiKey is 'string' and options.apiKey.length > 0
```

?

Reply:
It *can* be that but I've been using coffeescript-isms less as it makes switching back and forth harder - that said I'll admit I do have a soft spot for `and`/`or`